src/entities/elasticsearch.ts

```diff
@@ -126,7 +128,7 @@ export const defineElasticSteps = (
   When(
     'searching for',
     async (payload) => {
-      await request('POST', `${indexUri}/_flush`);
+      await request('POST', `${indexUri}/_refresh`);
```

Comment:
I think you can reuse the call to `refresh` here

Reply:
Currently `refresh()` is defined inside the scope of `create()`, so we can't use it here.